viewer/src/deps.cljs

```diff
@@ -0,0 +1,27 @@
+{:npm-deps
+ {"@codemirror/autocomplete" "^6.0.2"
```

Comment:
Based on my local install with these deps, on main I think `@nextjournal/lang-clojure` is still missing